<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QWidget" name="wgt_topLayer" native="true">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="7">
           <widget class="QPushButton" name="pb_max">
            <property name="text">
             <string>max</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lb_headicon">
            <property name="text">
             <string>headicon</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="pb_remote">
            <property name="text">
             <string>remote</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QPushButton" name="pb_min">
            <property name="text">
             <string>min</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lb_logo">
            <property name="text">
             <string>logo</string>
            </property>
           </widget>
          </item>
          <item row="0" column="8">
           <widget class="QPushButton" name="pb_close">
            <property name="text">
             <string>close</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="pb_setting">
            <property name="text">
             <string>setting</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pb_skin">
            <property name="text">
             <string>skin</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>500</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="wgt_middleLayer" native="true">
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QListWidget" name="lw_songNameList"/>
          </item>
          <item row="0" column="1">
           <widget class="QListWidget" name="lw_lyricShow"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QWidget" name="wgt_bottomLayer" native="true">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="12" rowspan="2">
           <widget class="QPushButton" name="pb_deskLyric">
            <property name="text">
             <string>deskLyric</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="13">
           <widget class="QSlider" name="hs_position">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="13">
           <widget class="QLabel" name="lb_position">
            <property name="text">
             <string>00:00/00:00</string>
            </property>
           </widget>
          </item>
          <item row="2" column="11">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="7">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3" rowspan="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>150</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="4" rowspan="2">
           <widget class="QPushButton" name="pb_prev">
            <property name="text">
             <string>prev</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" rowspan="2">
           <widget class="QLabel" name="lb_songName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>songName</string>
            </property>
           </widget>
          </item>
          <item row="2" column="10" rowspan="2">
           <widget class="QSlider" name="hs_volume">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="2" rowspan="2">
           <widget class="QLabel" name="lb_songicon">
            <property name="text">
             <string>songicon</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5" rowspan="2">
           <widget class="QPushButton" name="pb_play">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="6" rowspan="2">
           <widget class="QPushButton" name="pb_next">
            <property name="text">
             <string>next</string>
            </property>
           </widget>
          </item>
          <item row="2" column="13" rowspan="2">
           <widget class="QPushButton" name="pb_playMode">
            <property name="text">
             <string>playMode</string>
            </property>
           </widget>
          </item>
          <item row="2" column="8" rowspan="2">
           <widget class="QLabel" name="lb_volume">
            <property name="text">
             <string>volume</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
